informer/informer.go

```diff
@@ -195,7 +197,6 @@ func (i *Informer) Watch(ctx context.Context) (chan watch.Event, chan watch.Even
 				watchEventCounter.WithLabelValues("error").Inc()
 				errChan <- microerror.Mask(err)
 			}
-			close(i.initializer)
```

Comment:
This just got moved to `fillCache` where it is done via `defer` once the watcher got initialized.